File: symmetric_encryption/caesarCipher/caesar_cipher.cpp
#include "caesar_cipher.h"

std::string encryptCaesar(int key, std::string message) {
    std::string encryptedMessage = "";

    for(char letter : message) {
        encryptedMessage.push_back(static_cast<char>(static_cast<int>(letter) + key));
    }

    return encryptedMessage;
}

std::string decryptCaesar(int key, std::string message) {
    std::string decryptedMessage = "";

    for(char letter : message) {
        decryptedMessage.push_back(static_cast<char>(static_cast<int>(letter) - key));
    }

    return decryptedMessage;
}

File: symmetric_encryption/caesarCipher/caesar_cipher.py
chars : str = "abcdefghijklmnopqrstuvwxyz"

def krypter_caesar(message : str, key : int) -> str:
    encryptedMessage : str = ""

    for char in message:
        encryptedMessage += chars[chars.find(char.lower()) + key % 26]

    return encryptedMessage

def dekrypter_caesar(message : str, key : int) -> str:
    decryptedMessage : str = ""

    for char in message:
        decryptedMessage += chars[chars.find(char.lower()) - key % 26]

    return decryptedMessage
